flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -548,4 +567,10 @@
 
 	/** Not intended to be instantiated. */
 	private TaskManagerOptions() {}
+
+	/** Valid names of managed memory consumers. */
+	public static class ManagedMemoryConsumerNames {
```

Comment:
if not too late, anything against having enum instead of constants?

Reply:
It's to avoid converting from/to string type almost everywhere the names are used.